src/components/Picker/Picker.driver.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { PICKER_DATA_KEYS } from './dataHooks';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+
+export interface PickerDriver extends BaseUniDriver {
+  isArrowsSizeSet(arrowsSized): Promise<boolean>;
```

Comment:
We need to add `clickNext` and `clickPrev` or something like that, and also test the callbacks in the spec file

Reply:
I followed the TPA Dropdown click test. but I don't know if it's enough, let me know what you think